app-text/typst/typst-0.13.1.recipe

```diff
@@ -0,0 +1,55 @@
+SUMMARY="A typesetting system markup language"
+DESCRIPTION="Typst is a new markup-based typesetting system that is designed to be as powerful \
+as LaTeX while being much easier to learn and use. \
+Typst has:
+- Built-in markup for the most common formatting task
+- Flexible functions for everything else
+- A tightly integrated scripting system
+- Math typesetting, bibliography management, and more
+- Fast compile times thanks to incremental compilation
+- Friendly error messages in case something goes wrong"
+HOMEPAGE="https://github.com/typst/typst"
+COPYRIGHT="2022 Laurenz Mädje and Martin Haug."
+LICENSE="Apache v2"
+REVISION="1"; updateRevisionVariables
```

Comment:
What's the `updateeRevisionVariables` for here? First time I see it popping up. :)

Reply:
Don't know.

It's present in multiple recipe files I found in the haikuports source tree, and as I use one as a recipe template, I kept it as if it was some magic undocumented.

Removed.

Reply:
That is automatically generated by HaikuPorter in the `port.recipe` version in the work directory to have the revision available internally for the path variables like `$prefix` and all others which contains that. (`$prefix` is equivalent to `$(finddir B_PACKAGE_LINKS_DIRECTORY)/$portName-$portVersion-$REVISION/.self`) It should not be necessary to do that manually.

The main reason for https://github.com/haikuports/haikuporter/pull/323 was to avoid having to do something similar for `PACKAGE_NAME` and `PACKAGE_VERSION` by disallowing them for the main package, so they can always be taken from the recipe file name.